test/lib/rules/prefer-arrow-functions.js

```diff
@@ -40,12 +46,56 @@ tester.run('lib/rules/prefer-arrow-functions', rule, {
     {code: '["Hello", "World"].reduce(function(a, b) { return a + " " + b; })', errors: ['Prefer using arrow functions over plain functions']},
     {code: 'class obj {constructor(foo){this.foo = foo;}}; obj.prototype.func = function() {};', errors: ['Prefer using arrow functions over plain functions'], options: [{disallowPrototype:true}]},
     ...[
-        'function foo() { return 3; }',
-        'function foo(a) { return 3; }',
-        'var foo = function() { return "World"; }',
-        'var foo = function x() { return "World"; }',
-        'function foo(a) { /* yo */ return 3; }',
-        'function foo(a) { return a && (3 + a()) ? true : 99; }',
-    ].map(singleReturnOnly).map(testCase => Object.assign({errors: ['Prefer using arrow functions over plain functions which only return a value']}, testCase))
+      // Make sure it works with ES6 classes & functions declared in object literals
+      ['class MyClass { render(a, b) { return 3; } }', 'class MyClass { render = (a, b) => 3; }'],
```

Comment:
Ah, good point. We use https://babeljs.io/docs/plugins/transform-class-properties/

I'd be happy to add a setting `allowClassProperties` to the rule.